src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -340,17 +340,6 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     return $firewall;
                 })
             ->end()
-            ->validate()
```

Comment:
@chalasr this is now actually not needed anymore since triggering the deprecation is moved and is not triggered anymore at all if `stateless = true` or `security = false`.

Reply:
good catch!